target/linux/ramips/image/mt7621.mk

```diff
@@ -221,6 +221,20 @@ define Device/zbt-wg3526
 endef
 TARGET_DEVICES += zbt-wg3526
 
+define Device/hc5962
+  DTS := HC5962
+  BLOCKSIZE := 128KiB
+#  IMAGES := sysupgrade.bin
+  PAGESIZE := 2048
+  KERNEL_SIZE := 2097152
+#  FILESYSTEMS := squashfs
+  UBINIZE_OPTS := -E 5
+  IMAGE/sysupgrade.bin := append-kernel | pad-to $$(KERNEL_SIZE) | append-ubi | check-size $$$$(ralink_default_fw_size_32M)
```

Comment:
Drop this line and add the following instead:

```
IMAGE_SIZE := $(ralink_default_fw_size_32M)
IMAGE/sysupgrade.bin := sysupgrade-tar | append-metadata
```

That should create the correct the correct image format for sysupgrade.

If the sysupgrade.bin you created is intended for the first installation of LEDE (maybe via bootloader) use the following:

```
IMAGE_SIZE := $(ralink_default_fw_size_32M)
IMAGES += factory.bin
IMAGE/sysupgrade.bin := sysupgrade-tar | append-metadata
IMAGE/factory.bin := append-kernel | pad-to $$(KERNEL_SIZE) | append-ubi | check-size $$$$(IMAGE_SIZE)
```